storage/postgres/query_builder.go

```diff
@@ -103,6 +103,40 @@ func (pgq *pgQuery) List(ctx context.Context, entity PostgresEntity) (*sqlx.Rows
 	return pgq.db.QueryxContext(ctx, pgq.sql.String(), pgq.queryParams...)
 }
 
+func (pgq *pgQuery) Count(ctx context.Context, entity PostgresEntity) (int, error) {
+	if pgq.err != nil {
+		return 0, pgq.err
+	}
+
+	tableName := entity.TableName()
+	labelsEntity := entity.LabelEntity()
+
+	baseQuery := fmt.Sprintf("SELECT COUNT(*) FROM %s %s", tableName, mainTableAlias)
+
+	if labelsEntity != nil {
```

Comment:
Do we care about the labels when counting? Probably we do for visibilities as its often the case for a vis to have 10k labels - which brings another point - a default page size / max page size of 50 is suitable for most entities but probably not for lets say visibilities - so we might need a more sophisticated logic per entity for these sizes